apigee/analytics/analytics.go

```diff
@@ -16,96 +16,386 @@ package analytics
 
 import (
 	"bytes"
+	"compress/gzip"
+	"encoding/base64"
 	"encoding/json"
+	"errors"
 	"fmt"
+	"io/ioutil"
 	"net/http"
+	"os"
 	"path"
 	"time"
 
 	"github.com/apigee/istio-mixer-adapter/apigee/auth"
+	"github.com/google/uuid"
+	multierror "github.com/hashicorp/go-multierror"
+	"istio.io/istio/mixer/pkg/adapter"
 )
 
 const (
-	axPath       = "/axpublisher/organization/%s/environment/%s"
-	axRecordType = "APIAnalytics"
+	axRecordType        = "APIAnalytics"
+	defaultAnalyticsURL = "https://hybrid-eap.apigee.com/edgex/analytics"
```

Comment:
This doesn't make sense to me... We should be calling an endpoint on our internal edgemicro proxy and using the "apigee_base" config to derive that endpoint.

Reply:
OK so we'll need to update the edgemicro proxy to have this endpoint. Is that the code that is lost?